package/src/components/Attachment/Gallery.tsx

```diff
@@ -258,7 +258,7 @@ const GalleryWithContext = <
               <TouchableOpacity
                 activeOpacity={0.8}
                 disabled={preventPress}
-                key={`gallery-item-${url}/${rowIndex}/${images.length}`}
```

Comment:
Can't have `url` directly as part of key, since the signature keeps changing causing re-render of this component.